solr/CHANGES.txt

```diff
@@ -81,7 +81,9 @@ Upgrade Notes
 * SOLR-13985: Solr's Jetty now binds to localhost network interface by default for better out of the box security.
   Administrators that need Solr exposed more broadly can change the SOLR_JETTY_HOST property in their Solr include
   (solr.in.sh/solr.in.cmd) file. (Jason Gerlowski, David Smiley, Robert Muir)
-  
+
+* SOLR-14147: Solr now runs with the SOLR_SECURITY_MANAGER enabled by default. Administrators that need to run Solr with Hadoop will need to disable this feature via environment variable or in one of the Solr init scripts. Other features in Solr could also break. (Robert Muir, marcussorealheis) 
```

Comment:
I suggest to reword this slightly in the first sentence to a more generic description: "... now runs with java security manager enabled by default". This way users know what it does (even if they don't know that configuration parameter).

For the second sentence, I would tweak wording from "disable this feature via..." to "set   SOLR_SECURITY_MANAGER_ENABLED=false via...". It matches the actual variable name and gives them the little snippet they need, in case they need to do it.

Reply:
good point. I like that feedback

Reply:
Made the changes here.